support/docker-compose.yml

```diff
@@ -63,5 +63,17 @@ services:
     ports:
       - "8081:8081"
 
+  taurus:
+    image: ghcr.io/code0-tech/reticulum/ci-builds/taurus:${CI_PIPELINE_ID}
+    networks:
+      - boot
+    environment:
+      RABBITMQ_URL: nats://nats:4222
```

Comment:
This env var is implemented wrong in taurus. This needs to be updated to `NATS_URL`